ydb/core/viewer/protos/viewer.proto

```diff
@@ -375,6 +375,8 @@ message TTenant {
     uint64 StorageAllocatedLimit = 41;
     Ydb.Cms.DatabaseQuotas DatabaseQuotas = 42;
     repeated TStorageUsage StorageUsage = 43;
+    repeated TStorageUsage TabletStorage = 44;
```

Comment:
I think the idea was to make api where UI doesn't have to think. It just works with the only values it has.

Reply:
during the discussion we decided to leave the format with minor changes